tests/unit/pubsub.tcl

```diff
@@ -105,8 +105,8 @@ start_server {tags {"pubsub network"}} {
 
     test "UNSUBSCRIBE from non-subscribed channels" {
         set rd1 [valkey_deferring_client]
-        assert_equal {0 0 0} [unsubscribe $rd1 {foo bar quux}]
-
+        catch {unsubscribe $rd1 {foo bar quux}} e
+        assert_match {*Can't execute 'unsubscribe' command in non-subscribe mode*} $e
         # clean up clients
         $rd1 close
     }
```

Comment:
Maybe loop through all three [p|s]unsubscribe command in a single test?

Reply:
I combine the commands unsubscribe and punsubscribe in for loop, but for sunsubscribe  command,  I keep it in the pubsubshard.tcl file. Please take a look